testing/integration/vector_search_integration_test.py

```
@@ -675,6 +675,67 @@ def test_vector_search(self, config):
+        time.sleep(1)
+        # Points on the x-axis at increasing distance from the origin query.
+        # Insertion order (k0,k1,k2) is deliberately not distance order.
+        dist_by_key = {"k0": 3.0, "k1": 1.0, "k2": 2.0}
+        for key, d in dist_by_key.items():
+            v = np.zeros(dims, dtype=np.float32)
+            v[0] = d
+            self.valkey_conn.hset(key, mapping={"vec": v.tobytes()})
+        time.sleep(1)
+
+        query_vec = np.zeros(dims, dtype=np.float32).tobytes()
+        knn = f"*=>[KNN 3 @vec $b AS vscore]"
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)